.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,88 @@
+---
+name: NEURON Release
+about: Create a NEURON release for a new branch from master.
+title: 'NEURON [x.y.z] release'
+labels: 'release'
+assignees: ''
+
+---
+
+Action items
+============
+
+Pre-release
+---
+- [ ] Look out for ModelDB regressions by analyzing [nrn-modeldb-ci last version vs nightly reports](https://github.com/neuronsimulator/nrn-modeldb-ci/actions/workflows/nrn-modeldb-ci.yaml?query=event%3Aschedule++)
+- [ ] Create CoreNEURON release & update submodule
+
+Sanity checks
+---
+- [ ] Create `release/x.y` branch and make sure GitHub and Azure builds pass
+- [ ] Run [nrn-build-ci](https://github.com/neuronsimulator/nrn-build-ci/actions/workflows/build-neuron.yml) for the respective Azure build; see [Azure drop guide](https://github.com/neuronsimulator/nrn-build-ci#azure-wheels-testing---manual-workflow) 
+- [ ] Activate ReadTheDocs build for `release/x.y` & make it hidden. Check docs are fine after build is done.
+- [ ] Run BBP Simulation Stack & other relevant tests
+- [ ] Build release wheels but WITHOUT upload ([see details](https://nrn.readthedocs.io/en/latest/install/python_wheels.html#publishing-the-wheels-on-pypi-via-azure))
```

Comment:
By the way, we should avoid these http links and [add references to sections](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html) (because these links get broken often)

Reply:
Yeah, I agree in principle. But extra work require to maintain external links :) we should think about this aspect nevertheless